src/serviceContext.js

```diff
@@ -0,0 +1,177 @@
+const State = require('./state.js').State;
+const crypto = require('crypto');
+const KeyJar = require('../nodeOIDCMsg/src/oicMsg/keystore/KeyJar').KeyJar;
+const assert = require('chai').assert;
+
+/**
+ * @fileoverview This class keeps information that a client needs to be
```

Comment:
I don't think that the @fileoverview is needed for this file, as it only contains a single class. Instead I would suggest to make this the class comment.

Reply:
Fixed